### DISCUSSION TITLE
How can I edit the banner element

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#232] [Q&A] [answered]
Hello, somehow I'm completely on the hose right now.
I want to display an image in the layout manager. I made a new section, a new column and then a new element with a module inside. No problem.
Now I noticed that the banner element is already an image. How can I set it to display the banner?
I've read all the documentation, watched videos and searched on Google, nothing is described about the banner anywhere.
What is this banner element for and how can I configure it?
Many thanks for your help,
deltapapa

**Comment by Chacapamac** [comment]
Yeah, not to sure here, I try to add an image to the “Banner Section“ in the Layout Manager thinking that it will work in the frontend automatically, but nothing, no image is display. I’m not to sure how to use that “Banner Section“ element???

![banner section](https://user-images.githubusercontent.com/1907276/198838475-99116dcf-6933-469b-a37c-0d9b9cc128d7.jpg)

![Screen Shot 2022-10-29 at 10 58 45 AM](https://user-images.githubusercontent.com/1907276/198838487-6ee7929f-87b4-4d3f-9c99-67c6c1928a12.jpg)

**Comment by deltapapa01** [comment, marked as answer]
Hey and yay, I found it!
The banners can be controlled via the individual menu items, so you have the option of giving each menu item a separate banner.
I don't have a use for it right now, but it's a nice feature.
![image](https://user-images.githubusercontent.com/19833064/198953575-5db3e238-b349-470d-bc15-cd872a85b6db.png)

WBR from germany,
deltapapa
